docs/adr-0004-versioning.md

```diff
@@ -0,0 +1,75 @@
+# ADR 4: Versioning 5/1/2019
+## Context
+We build several different versions of Firefox for purposes such as release, beta testing, and staged rollout. It's not obvious how the version numbers are related to these types of builds, and features have sometimes disappeared in rollback versions, so we document them here to avoid confusion.
+
+Firefox TV releases are listed [https://github.com/mozilla-mobile/firefox-tv/releases](here).
```

Comment:
We should also mention the changelog and maybe that:
- Fire TV release includes LATs while the changelog does not
- changelog may include extra information (e.g. why is identical to the previous release)